docs/technical-documentation/migrate-7x.md

```diff
@@ -1,5 +1,6 @@
-## Migrating from Islandora 7.x to Islandora CLAW (YouTube Video)
+## Migrating from Islandora 7.x to Islandora 8 (YouTube Video)
 
-[![Migrating from Islandora 7.x to Islandora CLAW](https://img.youtube.com/vi/fEDzfSPjKEo/0.jpg)](https://www.youtube.com/watch?v=fEDzfSPjKEo)
+[![Islandora Webinar: Migrating from Islandora 7 to Islandora 8](https://img.youtube.com/vi/n0kLOB8ZdbM/0.jpg)](https://www.youtube.com/watch?v=n0kLOB8ZdbM)
+Nov 21, 2019
 
 For tools to migrate into Islandora 8 from an existing Islandora 7 instance, please see [migrate_7x_claw](https://github.com/Islandora-devops/migrate_7x_claw)
```

Comment:
While you're in here, seems like this should point to your new Migration Overview page instead or at least reference the CSV migration module.